.azure-pipelines/scripts/codeScan/pyspelling/inc_dict.txt

```diff
@@ -741,6 +743,7 @@ horovodrun
 hostfile
 Hounsfield
 howpublished
```

Comment:
which file used this word? shall we use full word for it?

Reply:
This word used here: 
![image](https://user-images.githubusercontent.com/51692656/206944001-58f29252-a84d-49c1-82cb-4dcdad0ba240.png)

Reply:
I will update this readme in the last sprint.